paper.md

```diff
@@ -300,7 +288,8 @@ This shows how Popper helps improve the performance of scientific workflows dras
 ### Workflow execution in Slurm clusters
 
 For this case study, we modified our training script to use the Horovod [@horovod] distributed deep learning framework to facilitate training with MPI [@gropp1999using] in a Slurm cluster.
-For running workflows in Slurm clusters, MPI supported container engines like Singularity, which is supported by popper need to be used.
+For running workflows in Slurm clusters, MPI supported container engines like Singularity, which is supported by Popper need to be used.
+The MPI based workflows can be made compatible between Slurm clusters with different MPI implementations if the Singularity images used in the workflows are built on the cluster by binding to the local MPI libraries.
```

Comment:
This also needs to explain what the contents of the image need to be (e.g. does any MPI version work?). Does the current implementation binds the drivers? Isn't the Slurm implementation using the [hybrid approach](https://sylabs.io/guides/3.6/user-guide/mpi.html#hybrid-model).

Reply:
Popper doesnt constrain on whether you need to use the bind model or hybrid model. As long as the MPI on slurm cluster and the mpi in the image is same things will work. Now this can be done in 2 ways.i) use an image  with the same mpi version as on the host ( this image might not be portable to other slurm clusters) ii) use the bind model to bind to whatever mpi version is available on the cluster to the image. (makes the image mpi version independent)